src/main/java/io/anserini/index/IndexCollection.java

```diff
@@ -148,6 +148,12 @@
 
     @Option(name = "-solr.zkChroot", usage = "the ZooKeeper chroot, if using a ZooKeeper URL instead of Solr")
     public String solrZkChroot = null;
+
+    // Note: This is used for ConcurrentSolrClient, where each processing thread has its own instance
+    // If the architecture is changed to share one ConcurrentSolrClient among all threads, the default should likely be increased
+    @Option(name = "-solr.client.threads", metaVar = "[Number]", required = false, usage = "Number of Threads for each SolrClient")
+    public int solrClientThreads = 1;
```

Comment:
Can we bump the default on this up to 16? https://github.com/castorini/Anserini/pull/573 will take us back to a single `ConcurrentSolrClient` among all threads.

Reply:
Seems like we're crossing streams with pull requests here. Yes, the default should be increased if the client is shared. 16 is not a bad value: Most Solr installations should handle that without problems.

Reply:
Yeah... might be easier if my PR is merged first than this one on top of it - fewer changes here.